docs/development.md

```diff
@@ -0,0 +1,133 @@
+﻿# FrankenPHP Development Environment Setup
+
+---
+
+## For Windows: WSL2 Setup
+
+1. Install WSL2:
```

Comment:
I suggest adding a little introduction here, to explain why WSL should be used etc.